src/modules/powerrename/PowerRenameUILib/Strings/en-us/Resources.resw

```diff
@@ -330,6 +330,12 @@
   <data name="FileCreationButton.[using:Microsoft.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
     <value>File creation date and time help</value>
   </data>
+  <data name="ErrorMessage_FileNameTooLong" xml:space="preserve">
+    <value>File name is too long</value>
+  </data>
+  <data name="ErrorMessage_InvalidChar" xml:space="preserve">
+    <value>File name contains invalid character(s): &gt;, &lt;, |, ", :, ?, *, \ or /</value>
```

Comment:
Windows Explorer uses a newline and no commas in the list - let's follow that:
![image](https://user-images.githubusercontent.com/65828559/185372456-4041c1d7-24a9-4b26-9c63-9a3e7c383e89.png)

Reply:
nice catch. fixed